core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -932,7 +932,7 @@ public static function addPageIcon($row, $label, DataContainer $dc=null, $imageA
 		$label = '<a href="' . self::addToUrl('pn=' . $row['id']) . '" title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['selectNode']) . '">' . $label . '</a>';
 
 		// Return the image
-		return '<a href="contao/main.php?do=feRedirect&amp;page=' . $row['id'] . '" title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['view']) . '" target="_blank">' . Image::getHtml($image, '', $imageAttribute) . '</a> ' . $label;
+		return '<a href="contao/preview.php?page=' . $row['id'] . '" title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['view']) . '" target="_blank">' . Image::getHtml($image, '', $imageAttribute) . '</a> ' . $label;
```

Comment:
There is actually a config value for `/preview.php`: `contao.previewScript`.

Reply:
That's the old legacy URL, which will be replaced automatically in the `Controller::replaceOldBePaths()` method:

https://github.com/contao/contao/blob/68b47c6ccab103c8fd205bc1c9a7f105c293d86b/core-bundle/src/Resources/contao/library/Contao/Controller.php#L1116-L1127

It has nothing to do with the `preview.php` entry point.

Reply:
Why do you use the legacy URL in favor of generating the `contao_backend_preview` route URL?

Reply:
Because when we start replacing the old BE URLs in the legacy code (which we should do at some point), we should replace them all instead of just one now.

Reply:
We should really stop that behavior of *don't refactor because we can't refactor it all just now* 🙈 🤷‍♂

Reply:
I disagree, because consistency is way more important IMHO.